workload/src/main/java/com/google/finapp/WorkloadMain.java

```diff
@@ -0,0 +1,154 @@
+                      CreateAccountRequest.Status.ACTIVE);
+          ids.add(response);
+        } catch (StatusRuntimeException e) {
+          numFailedCreateAccounts++;
+          logger.log(
+              Level.WARNING,
+              String.format("CreateAccount failed. Total count: %d", numFailedCreateAccounts));
+        }
+      }
+    }
+
+    void startSteadyLoad() {
+      Random random = new Random();
+      int numIds = ids.size();
+      if (numIds == 0) {
+        throw new IllegalStateException("No accounts were created successfully.");
+      }
+      while (true) {
+        ByteString fromId = ids.get(random.nextInt(numIds));
+        ByteString toId = ids.get(random.nextInt(numIds));
+        if (fromId.equals(toId)) {
+          continue;
+        }
+        WorkloadClient.getWorkloadClient(channel)
+            .moveAccountBalance(fromId, toId, DEFAULT_TRANSFER_AMOUNT);
+      }
+    }
+  }
+
+  public static void main(String[] args) {
+    CommandLine cmd = parseArgs(args);
+    String addressName = cmd.getOptionValue("a");
+    int port;
+    int numAccounts;
+    try {
+      port = ((Long) cmd.getParsedOptionValue("p")).intValue();
+      numAccounts = ((Long) cmd.getParsedOptionValue("n")).intValue();
+    } catch (ParseException e) {
+      throw new IllegalArgumentException("Input value cannot be parsed.", e);
+    }
+    ManagedChannel channel =
+        ManagedChannelBuilder.forAddress(addressName, port).usePlaintext().build();
+    WorkloadGenerator workloadGenerator = new WorkloadGenerator(channel);
+    workloadGenerator.seedData(numAccounts);
+    workloadGenerator.startSteadyLoad();
+  }
+
+  private static CommandLine parseArgs(String[] args) {
+    Options options = new Options();
+
+    options.addOption(
+        Option.builder("a")
+            .longOpt("address-name")
+            .desc("server address name")
+            .required(true)
+            .type(String.class)
+            .hasArg()
+            .build());
+
+    options.addOption(
+        Option.builder("p")
+            .longOpt("port")
+            .desc("server port")
+            .required(true)
+            .type(Number.class)
+            .hasArg()
+            .build());
+
+    options.addOption(
+        Option.builder("n")
+            .longOpt("num-accounts")
+            .desc("number of accounts to create")
+            .required(true)
+            .type(Number.class)
+            .hasArg()
+            .build());
+
+    CommandLineParser parser = new DefaultParser();
+    HelpFormatter formatter = new HelpFormatter();
+
+    try {
+      return parser.parse(options, args);
+    } catch (ParseException e) {
+      System.out.println(e.getMessage());
+      formatter.printHelp(
+          String.format(
+              "java -jar %s",
+              Paths.get("WorkloadMain.java").toAbsolutePath().normalize().toString()),
+          options);
+      System.exit(1);
+      return null;
```

Comment:
The program terminates before this with `System.exit`  but the `return null` is necessary for the code to compile.